src/common/telemetryHelper.ts

```diff
@@ -52,6 +52,13 @@ export class TelemetryHelper {
         Telemetry.send(event);
     }
 
+    public static sendDirect(eventName: string, properties?: Telemetry.ITelemetryProperties): void {
+        const event = TelemetryHelper.createTelemetryEvent(eventName, {isDirect: true});
+        if (properties)
+            TelemetryHelper.addTelemetryEventProperties(event, properties);

```

Comment:
Please, surround single-line "if" blocks with brackets, as well, because otherwise it could lead to hardly recognisable errors.
```js
        if (properties) {
            TelemetryHelper.addTelemetryEventProperties(event, properties);
        }
```